### PR TITLE
Bug Fix for ACS

### PR DESCRIPTION
**Pull request by rakshith91**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/17407

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py [hunk 1]
```diff
@@ -24,6 +24,12 @@ class SystemEventNames(str, Enum):
     AcsChatThreadCreatedWithUserEventName = (
         "Microsoft.Communication.ChatThreadCreatedWithUser"
     )
+    #backward compatibility
+    AcsChatParticipantAddedToThreadEvent = "Microsoft.Communication.ChatThreadParticipantAdded"
+    AcsChatParticipantRemovedFromThreadEvent = "Microsoft.Communication.ChatThreadParticipantRemoved"
+    # new constants matching the model names
+    AcsChatThreadParticipantAddedEventName = "Microsoft.Communication.ChatThreadParticipantAdded"
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py:31]: i'm inclined to add these additional constants to match the convention of other consts here - if there's a strong reason not to have two different consts with same value, i'm okay to remove it.

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py [hunk 2]
```diff
@@ -24,6 +24,12 @@ class SystemEventNames(str, Enum):
     AcsChatThreadCreatedWithUserEventName = (
         "Microsoft.Communication.ChatThreadCreatedWithUser"
     )
+    #backward compatibility
+    AcsChatParticipantAddedToThreadEvent = "Microsoft.Communication.ChatThreadParticipantAdded"
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py:28]: If you did miss them in the first release, don't add them. You don't have to fix something that didn't exists :)

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py [hunk 3]
```diff
@@ -24,6 +24,12 @@ class SystemEventNames(str, Enum):
...
+    AcsChatParticipantRemovedFromThreadEvent = "Microsoft.Communication.ChatThreadParticipantRemoved"
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py:29]:
```suggestion
```

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py [hunk 4]
```diff
@@ -24,6 +24,10 @@ class SystemEventNames(str, Enum):
     AcsChatThreadCreatedWithUserEventName = (
         "Microsoft.Communication.ChatThreadCreatedWithUser"
     )
+    #backward compatibility
+    # new constants matching the model names
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py:28]:
```suggestion
```

**Review by lmazuel — APPROVED** [review]

File: sdk/eventgrid/azure-eventgrid/tests/test_serialization.py [hunk 5]
```diff
@@ -114,3 +114,7 @@ def test_import_from_sytem_events(self):
         assert SystemEventNames.KeyVaultKeyNearExpiryEventName == "Microsoft.KeyVault.KeyNearExpiry"
         var = SystemEventNames.ServiceBusActiveMessagesAvailableWithNoListenersEventName
         assert var == "Microsoft.ServiceBus.ActiveMessagesAvailableWithNoListeners"
+        var = SystemEventNames.AcsChatParticipantAddedToThreadEvent
+        assert var == "Microsoft.Communication.ChatThreadParticipantAdded"
+        var = SystemEventNames.AcsChatParticipantRemovedFromThreadEvent
+        assert var == "Microsoft.Communication.ChatThreadParticipantRemoved"
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/tests/test_serialization.py:120]:
```suggestion
        var = SystemEventNames.AcsChatThreadParticipantAddedEventName
        assert var == "Microsoft.Communication.ChatThreadParticipantAdded"
        var = SystemEventNames.AcsChatThreadParticipantRemovedEventName
        assert var == "Microsoft.Communication.ChatThreadParticipantRemoved"
```
```suggestion
        var = SystemEventNames.AcsChatParticipantAddedToThreadEvent
        assert var == "Microsoft.Communication.ChatThreadParticipantAdded"
        var = SystemEventNames.AcsChatParticipantRemovedFromThreadEvent
        assert var == "Microsoft.Communication.ChatThreadParticipantRemoved"
```

**Pull request merged**